grails-app/views/returns/show.gsp

```diff
@@ -130,7 +130,12 @@
                                 <g:message code="stockMovement.status.label"/>
                             </td>
                             <td class="value">
-                                <format:metadata obj="${stockMovement?.shipment?.status?:stockMovement?.statusCode}"/>
+                                <g:if test="${stockMovement?.shipment?.status}">
```

Comment:
@drodzewicz Is there a reason behind this change into `if-else`? You could add the `?.code` to the first part and have the same result:
```<format:metadata obj="${stockMovement?.shipment?.status?.code ?: stockMovement?.statusCode}"/>```